pkg/azurefile/nodeserver.go

```diff
@@ -95,6 +99,42 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 				return nil, status.Errorf(codes.InvalidArgument, "invalid mountPermissions %s", perm)
 			}
 		}
+		if d.enableKataCCMount {
```

Comment:
as long as `enableKataCCMount` is enabled, e2e tests would fail with following error without katacc configured, is it possible to add a new parameter in storage class or pv, then it only fails when enableKataCCMount is specified by the user, I think that's more reasonable. Otherwise if we `enableKataCCMount` in driver level on aks, this csi driver would fail on all nodes without katacc configured

```
I0904 20:54:51.842457       1 request.go:973] Error in request: resource name may not be empty
W0904 20:54:51.842481       1 utils.go:335] Failed to get runtimeClass : resource name may not be empty
E0904 20:54:51.842504       1 utils.go:106] GRPC error: rpc error: code = Internal desc = failed to check if runtime class  is confidential: resource name may not be empty
```

Reply:
related azure file csi driver logs: https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_azurefile-csi-driver/1971/pull-azurefile-csi-driver-e2e-capz/1831414149721100288/artifacts/clusters/capz-k7plfw/kube-system/csi-azurefile-node-vcb86/azurefile.log